readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -264,7 +258,6 @@ def append_conf(self):
         )
 
     def build(self):
-        self.clean()
```

Comment:
I know this was probably not needed, but I'd like to reduce moving pieces here and not try to "clean up" too much and randomly break things when we're already doing a large refactor.

Reply:
I removed it because there is no "old artifact path" (variable `self.old_artifact_path`) to remove anymore. This is because we only have one output directory now (`_readthedocs/<format>`). Note the super `BaseBuilder.clean()` method was completely removed.